lstchain/io/dataselection.py

```diff
@@ -0,0 +1,263 @@
+from ctapipe.core import Component
+from ctapipe.core.traits import Int, Float, List
+import numpy as np
+import astropy.units as u
+from pyirf.binning import create_bins_per_decade  # , add_overflow_bins
+
+__all__ = ["DataSelection", "DataBinning"]
+
+
+class DataSelection(Component):
+    """
+    Collect various selection cuts to be applied for IRF production and
+    DL3 data reduction
+
+    Parameters for event filters will be combined in a dict so that the
+    filter_events() can be used.
+    """
+
+    intensity = List(
+        help="Range of intensity of event filter",
+        trait=Float(),
+        default_value=[0, np.inf],
+    ).tag(config=True)
+
+    length = List(
+        help="Range of length for event filter",
+        trait=Float(),
+        default_value=[0, np.inf],
+    ).tag(config=True)
+
+    width = List(
+        help="Range of width for event filter",
+        trait=Float(),
+        default_value=[0, np.inf],
+    ).tag(config=True)
+
+    r = List(
+        help="Range of r for event filter",
+        trait=Float(),
+        default_value=[0, 1],
+    ).tag(config=True)
+
+    wl = List(
+        help="Range of wl for event filter",
+        trait=Float(),
+        default_value=[0.01, 1],
+    ).tag(config=True)
+
+    leakage_intensity_width_2 = List(
+        help="Range for leakage_intensity_width_2 of event filter",
+        trait=Float(),
+        default_value=[0, 1],
+    ).tag(config=True)
+
+    fixed_gh_cut = Float(
+        help="Fixed selection cut for gh_score (gammaness)",
+        default_value=0.5,
+    ).tag(config=True)
+
+    fixed_theta_cut = Float(
+        help="Fixed selection cut for theta",
+        default_value=0.5,
+    ).tag(config=True)
+
+    fixed_source_fov_offset_cut = Float(
+        help="Fixed selection cut for source FoV offset",
+        default_value=3,
+    ).tag(config=True)
+
+    src_dep_alpha = Float(
+        help="Selection cut for source dependent parameter - alpha",
+        default_value=8.0,
+    ).tag(config=True)
+
+    lst_tel_ids = List(
+        help="List of selected LST telescope ids",
+        trait=Int(),
+        default_value=[1],
+    ).tag(config=True)
+
+    magic_tel_ids = List(
+        help="List of selected MAGIC telescope ids",
+        trait=Int(),
+        default_value=[1, 2],
+    ).tag(config=True)
+
+    aliases = {
```

Comment:
Components do not have aliases. Only `Tools`, since aliases define the command line options.

Reply:
Ah ok! I will remove them.